cli/pkg/kctrl/cmd/core/namespace_flags.go

```diff
@@ -26,6 +32,7 @@ func (s *NamespaceFlags) SetWithPackageCommandTreeOpts(cmd *cobra.Command, flags
 	namespaceEnvVariableKey := fmt.Sprintf("%s_NAMESPACE", strings.ToUpper(opts.BinaryName))
 	name := flagsFactory.NewNamespaceNameFlag(&s.Name, namespaceEnvVariableKey)
 	cmd.Flags().VarP(name, "namespace", "n", fmt.Sprintf("Specified namespace ($%s or default from kubeconfig)", namespaceEnvVariableKey))
+	cmd.Flags().StringArrayVar(&s.AllowedSharedNamespaces, "dangerous-allow-use-of-shared-namespace", []string{}, "Allow use of shared namespaces (optional, comma separated strings)")
```

Comment:
I am thinking if this should be a boolean 🤔

Reply:
The idea is to allow the user to be specific about which ns they want to allow installation in 🤔 
I can see it being a boolean because we are just protecting two namespaces anyway

Reply:
Adopted!